scripted/src/locales/el/locale.js

```diff
@@ -0,0 +1,253 @@
+
+    "%export.exportButtonLabel": "Εξαγωγή",
+    "%export.exportAllButtonLabel": "Εξαγωγή όλων",
+    "%export.exportDialogBoxCloseButtonLabel" : "Κλείσιμο",
+    "%export.exportDialogBoxPrompt": "Αντιγράψτε αυτόν τον κώδικα στο πρόχειρό σας, όπως θα αντιγράφατε κάθε κείμενο. Πατήστε ESC για να κλείσετε αυτό το παράθυρο.",
+    "%export.focusDialogBoxCloseButtonLabel": "Κλείσιμο",
+    "%export.rdfXmlExporterLabel": "RDF/XML",
+    "%export.smwExporterLabel": "Semantic wikitext",
+    "%export.exhibitJsonExporterLabel": "Exhibit JSON",
+    "%export.tsvExporterLabel": "Τιμές διαχωρισμένες με κόμμα",
+    "%export.htmlTableExporterLabel": "HTML Table",
+    "%export.htmlExporterLabel": "HTML αυτής της προβολής",
+    "%export.bibtexExporterLabel": "BibTeX",
+
+    "%import.couldNotLoad": "Αδυναμία φόρτωσης δεδομένων από το αρχείο %1$s στη βάση δεδομένων",
+    "%import.couldNotParse": "Αδυναμία ανάλυσης %1$s",
+    "%import.missingOrFilesystem": "Αδυναμία πρόσβασης στο αρχείο %1$s, πιθανόν γιατί το αρχείο λείπει ή επειδή έχετε πρόσβαση στα αρχεία σας μέσω ενός συστήματος αρχείων αντί ενός webserver ενώ χρησιμοποιείτε Chrome or IE. Χρησιμοποιήστε διαφορετικό περιηγητή ή μετακινήστε τα αρχεία σας σε έναν webserver.",
+    "%import.httpError": "Αποτυχία πρόσβασης στο αρχείο %1$s (HTTP %2$d)",
+    "%import.failedAccess": "Αποτυχία πρόσβασης στο αρχείο %1$s%2$s",
+    "%import.failedAccessHttpStatus": " (HTTP %1$d)",
+    "%import.missingFragment": "Αποτυχία εντοπισμού αποσπάσματος στο αρχείο %1$s",
+
+    "%database.itemType.label": "Καταχώριση",
+    "%database.itemType.pluralLabel": "Καταχωρίσεις",
+    "%database.labelProperty.label": "ετικέτα",
+    "%database.labelProperty.pluralLabel": "ετικέτες",
+    "%database.labelProperty.reverseLabel": "ετικέτα του",
+    "%database.labelProperty.reversePluralLabel": "ετικέτες του",
+    "%database.idProperty.label": "id",
+    "%database.idProperty.pluralLabel": "ids",
+    "%database.idProperty.reverseLabel": "id του",
+    "%database.idProperty.reversePluralLabel": "ids των",
+    "%database.typeProperty.label": "είδος",
+    "%database.typeProperty.pluralLabel": "είδη",
+    "%database.typeProperty.reverseLabel": "είδος του",
+    "%database.typeProperty.reversePluralLabel": "είδη των",
+    "%database.uriProperty.label": "URI",
+    "%database.uriProperty.pluralLabel": "URIs",
+    "%database.uriProperty.reverseLabel": "URI του",
+    "%database.uriProperty.reversePuralLabel": "URIS των",
+    "%database.sortLabels.text.ascending":  "α - ω",
+    "%database.sortLabels.text.descending": "ω - α",
+    "%database.sortLabels.number.ascending":  "το μικρότερο πρώτα",
+    "%database.sortLabels.number.descending": "το μεγαλύτερο πρώτα",
+    "%database.sortLabels.date.ascending":  "το αρχικό πρώτα",
+    "%database.sortLabels.date.descending": "το τελευταίο πρώτα",
+    "%database.sortLabels.boolean.ascending":  "το λάθος πρώτα",
+    "%database.sortLabels.boolean.descending": "το σωστό πρώτα",
```

Comment:
Δεν είμαι σίγουρη ότι βγάζει νόημα η μετάφραση του true/false ως λάθος/σωστό in context. Η επίσημη μετάφραση είναι αληθές/ψευδές, αλλά μήπως αυτό είναι too technical?